.sonarcloud.properties

```diff
@@ -2,7 +2,7 @@ sonar.projectName=lear
 sonar.projectVersion=Autoscan
 
 # Path to sources
-sonar.sources=colin-api/src,legal-api/src,data-reset-tool/src,jobs/**/src,queue_services/**/src
+sonar.sources=colin-api/src/**/*,legal-api/src/**/*,data_reset_tool/src/**/*,jobs/**/*,queue_services/**/src/**/*
```

Comment:
Do the test paths need to be updated similarly?

Reply:
Yes, I should. Struggling with making this work due to the layered directories.